.ci/pipeline_definitions

```diff
@@ -1,24 +1,12 @@
 gardener-website-generator:
-  template: 'default'
   base_definition:
-    traits:
-      version: ~
-      scheduling:
-        suppress_parallel_execution: true
-    repo:
-      name: 'gardener_website_generator'
-      trigger: true
-      cfg_name: 'github_com'
     repos:
     - name: 'gardener_website'
       path: 'gardener/website'
       branch: 'master'
-      cfg_name: 'github_com'
-      trigger: false
```

Comment:
this is a change in behaviour as this attributes controls whether a pipeline build should be scheduled in case there was a repository update.
with this change you would also schedule pipeline builds on gardener/website head update.

Reply:
Based on https://gardener.github.io/cc-utils/pipeline_job.html#default-behaviour-for-trigger-attribute isn't here the default value false though?

Reply:
my bad, confused `repo` and `repos`
yes, you are right. `false` is the default value for additional repositories (-> `repos` attribute)